src/lib.rs

```diff
@@ -135,3 +137,4 @@ mod diff;
 mod assert;
 pub use assert::Assert;
 pub use assert::OutputAssertionBuilder;
+pub use environment::Environment;
```

Comment:
Do you want to add some specific docs here mentioning this is a report from the external crate? I assume it inherits the docs from the item in the crate right now.